packages/main/src/ColorPalette.ts

```diff
@@ -168,7 +168,8 @@ class ColorPalette extends UI5Element {
 	_itemNavigationRecentColors: ItemNavigation;
 	_recentColors: Array<string>;
 	moreColorsFeature?: ColorPaletteMoreColors;
-	_currentlySelected?: ColorPaletteItem;
+	_currentlySelected?: ColorPaletteItem | null;
+	_shouldFocusRecentColors = false;
```

Comment:
Remove value assign from here and assign the value in the constructor. Also add type for the variable.